packages/account/src/Sections/Profile/PersonalDetails/personal-details-form.tsx

```diff
@@ -0,0 +1,714 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Formik, Form, FormikHelpers } from 'formik';
+import { BrowserHistory } from 'history';
+import { withRouter } from 'react-router';
+import {
+    Button,
+    Checkbox,
+    DesktopWrapper,
+    Dropdown,
+    FormSubmitErrorMessage,
+    HintBox,
+    Input,
+    Loading,
+    MobileWrapper,
+    SelectNative,
+    Text,
+} from '@deriv/components';
+import { GetSettings } from '@deriv/api-types';
+import { WS, getBrandWebsiteName, routes, useIsMounted } from '@deriv/shared';
+import { Localize, localize } from '@deriv/translations';
+import { observer, useStore } from '@deriv/stores';
+import LeaveConfirm from 'Components/leave-confirm';
+import FormFooter from 'Components/form-footer';
+import FormBody from 'Components/form-body';
+import { DateOfBirthField } from 'Components/forms/form-fields';
+import FormSubHeader from 'Components/form-sub-header';
+import LoadErrorMessage from 'Components/load-error-message';
+import POAAddressMismatchHintBox from 'Components/poa-address-mismatch-hint-box';
+import { getEmploymentStatusList } from 'Sections/Assessment/FinancialAssessment/financial-information-list';
+import InputGroup from './input-group';
+import { getPersonalDetailsInitialValues, getPersonalDetailsValidationSchema, makeSettingsRequest } from './validation';
+import FormSelectField from 'Components/forms/form-select-field';
+
+type TRestState = {
+    show_form: boolean;
+    errors?: boolean;
+    api_error?: string;
+    changeable_fields?: string[];
+    form_initial_values?: Record<string, any>;
+};
+
+export const PersonalDetailsForm = observer(({ history }: { history: BrowserHistory }) => {
+    const [is_loading, setIsLoading] = React.useState(true);
+    const [is_state_loading, setIsStateLoading] = React.useState(false);
+    const [is_btn_loading, setIsBtnLoading] = React.useState(false);
```

Comment:
**NIT**: Lets use Formik states for this while refactoring

Reply:
Should I remove it on the account-V2 package?

Reply:
Yeh. This is just a note to remember. U can use formik states to handle button loader states